spec.emu

```diff
@@ -286,7 +286,7 @@ contributors: Ujjwal Sharma, Younies Mahmoud
           1. If _value_ is 0 and _display_ is `"auto"`, then
             1. Skip to the next iteration.
           1. Let _nf_ be ! Construct(%NumberFormat%, &laquo; _durationFormat_.[[Locale]], _nfOpts_ &raquo;).
-          1. Let _num_ be ! FormatNumeric(_durationFormat_.[[NumberFormat]], 𝔽(_value_)).
+          1. Let _num_ be ! FormatNumeric(_nf_, 𝔽(_value_)).
```

Comment:
I suppose I missed this when I moved from a single (cached) numberformat instance to dynamic one based on the style.